test/controllers/ReviewPenaltyController.test.ts

```diff
@@ -72,7 +72,9 @@ describe('ReviewPenaltyController', () => {
 
     });
 
-    it('should go to other reasons disclaimer screen when continue is pressed', async () => {
+    it('should go to Choose Reason screen when continue is pressed', async () => {
```

Comment:
That test should not be changed because a) story does not define navigation logic b) existing logic should be preserved as default and tested.

If conditional navigation was to be implemented it would require two tests, one for feature flag on and other for feature flag off.

Reply:
As mentioned in previous comment, I can't figure out how to complete the story without extending the SafeNavigationBaseController. With this in mind, I can't access the page without first hitting Continue on a previous page, which according to the Prototype is this one, hence the navigation update. 

I will however get the second condition tested.

Reply:
Resolved: Navigation is no longer affected by this feature so the test has been unaltered.